pages/about.html

```diff
@@ -4,9 +4,53 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Awesome Sunny</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header> 
+    <h1 class="page-titles">About Sunny</h1>
+    <nav>
+      <ul class="container">
+        <li>
+          <a href = "portfolio.html">Portfolio</a> 
+        </li>
+        <li>
+          <a href = "index.html"> Index</a>
```

Comment:
Also my HTML validator caught these spaces around the `=` in some of your attributes, which is not allowed. So this tag for example should be `<a href="index .html">`. I'll skip pointing it out the other times, this and the trailing spaces were the only issues the validator caught!